README.md

```diff
@@ -1,24 +1,172 @@
+## 🛠 Built With <a id="built-with"></a>
 
-* Database initialization
+### Tech Stack <a id="tech-stack"></a>
 
-* How to run the test suite
+> Ruby on Rails
+> HTML
+> CSS
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML, CSS, Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">Postgresql</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+- List all the articles in the blog
+- List all the articles of a given user
+- A user can create a new article
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+
+- Live demo pending
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+> Ruby 3.2 installed in your machine
+### Setup
+
+Clone this repository to your desired folder:
+
+> git clone https://github.com/itswali/rails_blog_app
+
+### Install
+
+Install this project with:
+
+bundle install
+
+### Usage
+
+rails s and open http://localhost:3000
+
+
+### Run tests
+
+To run tests, run the following command:
+
+rspec
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a id="authors"></a>
+
+
+👤 **Wali Muhammad**
+
+* GitHub: [@Itswali](https://github.com/Itswali)
+* Twitter: [@Wali-Muhammad](https://twitter.com/WaliMuh94818599)
+* LinkedIn: [@wali-Muhammad](https://linkedin.com/in/wali-muhammad-666040244/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a id="future-features"></a>
+
+
+- Nicer styling is going to be implemented
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Itswali/rails_blog_app/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a id="support"></a>
+
+If you like this project please star this repo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a id="acknowledgements"></a>
+
+I would like to thank microverse inc. for helping me in my professional carreer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a id="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Please resolve the URL for the link here so it links properly to a license file in order to make your documentation complete.

Reply:
Location for the MIT Lincense file has been updated. 

Thanks For Pointing out the error.